tests/ui/absurd-extreme-comparisons.stderr

```diff
@@ -1,5 +1,5 @@
 error: this comparison involving the minimum or maximum element for this type contains a case that is always true or always false
-  --> $DIR/absurd-extreme-comparisons.rs:14:5
+  --> tests/ui/absurd-extreme-comparisons.rs:14:5
```

Comment:
Those changes look like they will cause trouble in the Rust repo. :thinking: I haven't tested that yet, though.

Reply:
I changed it so it doesn't show the tests/ui part, but it still shows the subdirectories within that